tidy3d/components/autograd/utils.py

```diff
@@ -0,0 +1,15 @@
+# utilities for working with autograd
+
+import typing
+
+from autograd.tracer import getval
+
+
+def get_static(x: typing.Any) -> typing.Any:
```

Comment:
Is this really necessary? Why don't we just use `getval` directly? If it's because of naming, we could also `from autograd.tracer import getval as get_static` here?